### PR TITLE
Add debug logs to test issue with post execute deployment proxy

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

**Comment by EarthmanT** [comment]
Do you think we should handle the error in the operation or inside of the [client call handler](https://github.com/cloudify-incubator/cloudify-utilities-plugin/blob/0517aa0a3f71a9c090aba2671db35fdda649fac9/cloudify_deployment_proxy/__init__.py#L110)?

**Comment by mabuaisha** [comment]
@EarthmanT I think the error which we have right now is come while trying to process the execution list for the deployment proxy and I think the error related to data issue and it could be the fix inside the client call handler, but need to figure out what is the main reason for causing the error I got

File: cloudify_deployment_proxy/__init__.py [hunk 1]
```diff
@@ -304,13 +311,23 @@ def post_execute_node_instance_proxy(self):
         return True
 
     def post_execute_deployment_proxy(self):
+        runtime_prop = ctx.instance.runtime_properties['deployment']
+        ctx.logger.debug(
+            'Runtime  deployment properties {}'.format(runtime_prop))
```
Comment (EarthmanT) [inline cloudify_deployment_proxy/__init__.py]: `{0}`

File: cloudify_deployment_proxy/__init__.py [hunk 2]
```diff
@@ -304,13 +311,23 @@ def post_execute_node_instance_proxy(self):
...
 
         if 'outputs' \
                 not in ctx.instance.runtime_properties['deployment'].keys():
             update_attributes('deployment', 'outputs', dict())
+            ctx.logger.debug('No deployment proxy outputs exist !!!!')
```
Comment (EarthmanT) [inline cloudify_deployment_proxy/__init__.py]: Can we do with a `.` instead of ` !!!!` :)

File: cloudify_deployment_proxy/__init__.py [hunk 3]
```diff
@@ -304,13 +311,23 @@ def post_execute_node_instance_proxy(self):
...
 
         try:
+            ctx.logger.debug('Deployment Id is {}'.format(self.deployment_id))
```
Comment (EarthmanT) [inline cloudify_deployment_proxy/__init__.py]: `{0}`

File: cloudify_deployment_proxy/__init__.py [hunk 4]
```diff
@@ -304,13 +311,23 @@ def post_execute_node_instance_proxy(self):
...
+
             response = self.client.deployments.outputs.get(self.deployment_id)
+
+            ctx.logger.debug(
+                'Deployment outputs response  {}'.format(response))
```
Comment (EarthmanT) [inline cloudify_deployment_proxy/__init__.py]: `{0}`

File: cloudify_deployment_proxy/__init__.py (hunk 2 above)
Comment (mabuaisha) [reply cloudify_deployment_proxy/__init__.py]: done

File: cloudify_deployment_proxy/__init__.py (hunk 3 above)
Comment (mabuaisha) [reply cloudify_deployment_proxy/__init__.py]: done

File: cloudify_deployment_proxy/__init__.py (hunk 4 above)
Comment (mabuaisha) [reply cloudify_deployment_proxy/__init__.py]: done

**Pull request merged**